PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -424,6 +424,7 @@ protected function do_close()
         $this->frame_queue = new \SplQueue();
         $this->method_queue = [];
         $this->setIsConnected(false);
+        $this->markChannelsClosed();
```

Comment:
word `mark` is confusing and seems we are faking something. Why not just `closeChannels()` ?

Reply:
We already have `closeChannels()`, but we can't use it because we need to close "silently", without sending anything.